mod/researched_metrics/set/type/relationship_answer.rb

```diff
@@ -131,8 +131,8 @@ def legend
                   value: card.value,
                   import: card.imported?,
                   comments: field_nest(:discussion, view: :core),
-                  subject_company: card.company,
-                  object_company: card.related_company
+                  subject_company: Card.fetch_name(card.company),
+                  object_company: Card.fetch_name(card.related_company)
```

Comment:
hack to fix non-standard names in JSON.  Would prefer to pull these from relationships table.